devolutions-session/src/dvc/fs.rs

```diff
@@ -0,0 +1,38 @@
+use std::path::PathBuf;
+
+pub struct ProcessTmpFile(PathBuf);
+
+impl ProcessTmpFile {
+    pub fn new(extension: &str) -> anyhow::Result<Self> {
+        // Create temporary batch file
```

Comment:
style: Per IronRDP STYLE.md

```suggestion
        // Create temporary batch file.
```